.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: java
 dist: trusty
 before_install:
+- rm -rf $HOME/.m2/repository/io/vlingo
```

Comment:
@pflueras What will this do when run on Travis-CI server? I mean here:

https://travis-ci.org/vlingo/vlingo-platform

Reply:
This just cleans up vlingo dependencies from local maven repo in Travis CI.
However this live was added and removed during commits, therefore it's not present anymore here.